package.json

```diff
@@ -91,6 +91,13 @@
     "build:product": "ng build product --prod",
     "release:organization:with-changelog": "cd feature-libs/organization && release-it && cd ../..",
     "release:product:with-changelog": "cd feature-libs/product && release-it && cd ../..",
+    "build:misc": "yarn --cwd feature-libs/misc/storefinder run build:schematics && ng build misc --prod",
+    "release:misc:with-changelog": "cd feature-libs/misc && release-it && cd ../..",
```

Comment:
we've dropped `misc` lib in favour of `storefinder` in `develop` . Can we remove them in this PR?

Reply:
Removed